bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableClient.scala

```diff
@@ -190,6 +190,54 @@ case object ConeSearchBackend extends CachedBackend with RemoteCallBackend {
   override def queryUrl(e: SearchKey): String = s"${e.url}/cgi-bin/conesearch.py"
 }
 
+case object GaiaBackend extends CachedBackend with RemoteCallBackend {
```

Comment:
The substantive part of the PR is the `GaiaBackend`, which despite the name is charged with forming the query to send to the Gaia server.

None of the "backend" options are really unit tested, presumably because that would require making remote calls. I'm not sure what to do for testing other than monkey testing.

Reply:
If I recall this, it was possible to do some testing using local files